warehouse/tuf/tasks.py

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tuf import repository_tool
+
+from warehouse.tasks import task
+from warehouse.tuf.interfaces import IKeyService
+
+TOPLEVEL_ROLES = ["root", "snapshot", "targets", "timestamp"]
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
```

Comment:
Will we ever be invoking this as a Celery task, or is this just being implemented to match other CLI commands that call tasks?

Reply:
The latter -- we shouldn't ever need to run this either manually or periodically after initial setup. I did it as a Celery task to get access to a `request` object; is there another way to do that just in the context of the CLI?

Reply:
Gotcha, then I think you can use the `warehouse.cli.warehouse.command` decorator to receive a `config` object on your CLI entry point function instead. See [the shell cli command](https://github.com/pypa/warehouse/blob/f12bcb4bcfa557fb5a3a0a50a35ab52163db5491/warehouse/cli/shell.py#L60) for example.